src/xcode/ENA/ENA/Source/Workers/Store/Models/EUTracingSettings.swift

```diff
@@ -0,0 +1,30 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+struct EUTracingSettings: Codable {
+	let isAllCountriesEnbled: Bool
+
+	/// List of country identifier. Equals the initializing country code.
+	let enabledCountries: [String]
+
+	init(isAllCountriesEnbled: Bool = false, enabledCountries: [String] = []) {
```

Comment:
Hmm isn't there a `Country` coming up in this PR: https://github.com/corona-warn-app/cwa-app-ios/pull/1121 Maybe we should wait for this? 🤔

Reply:
You are right. Only the IDs (country code) of the selected countries are persisted. And later we can create a Country object out of this ID.